### PR TITLE
test: add mint nft action cypress

### PR DESCRIPTION
**Pull request by olavoparno**
#569

closes 

fixes 

## What changed? Why?

Introduced LocalWallet for cypresses tests to mock a temporary authentication and added a test flow for the mint nft user action

## How has it been tested?

- [X] Locally
- [ ] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

### CODE REVIEW

File: cypress/fixture/mocks.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const mockWallet = {
+  password: Math.random().toString(36).substring(7),
```
Comment (lucasrmp-cb) [inline cypress/fixture/mocks.ts:2]: Can we use faker here?

File: cypress/fixture/mocks.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+}
+
+export const mockRandomUser = {
+  email: `johndoe${Math.random().toString(36).substring(7)}@email.com`,
```
Comment (lucasrmp-cb) [inline cypress/fixture/mocks.ts:6]: +1 on the faker question

File: src/components/app/updateUserProfileForm/step1/control.tsx [hunk 3]
```diff
@@ -125,7 +125,7 @@ export function UpdateUserProfileForm({
                 <FormItem>
                   <FormLabel>Email</FormLabel>
                   <FormControl>
-                    <Input placeholder="Your email" {...field} />
+                    <Input data-testid="email-address-input" placeholder="Your email" {...field} />
```
Comment (lucasrmp-cb) [inline src/components/app/updateUserProfileForm/step1/control.tsx:128]: Are you using the `data-testid`'s you added in this PR? Not seeing any references

File: cypress/fixture/mocks.ts (hunk 1 above)
Comment (olavoparno) [reply cypress/fixture/mocks.ts:2]: Definitely! Thanks for the suggestion, I wasn't aware we used

File: src/components/app/updateUserProfileForm/step1/control.tsx (hunk 3 above)
Comment (olavoparno) [reply src/components/app/updateUserProfileForm/step1/control.tsx:128]: You're right, I'm not, at least for this field. Thanks for noticing

**Review by lucasrmp-cb — APPROVED** [review]

**Pull request merged**